### ISSUE TITLE
EOF error in hstream.moe, is there a way to fix it or bypass it?

### ISSUE THREAD

**Issue by Anonym121** [gan-of-culture/get-sauce#151]
**Describe the bug**
When trying to get some videos in hstream.moe such as this one: https://hstream.moe/hentai/kansen-inyoku-no-rensa-1

get-sauce displays the following error:
2024/08/18 23:23:07 Get "https://tenka-str.ane-h.xyz/2023/Kansen.Inyoku.no.Rensa/E01/2160/manifest.mpd": EOF: https://hstream.moe/hentai/kansen-inyoku-no-rensa-1

**To Reproduce**
1. Execute get-sauce https://hstream.moe/hentai/kansen-inyoku-no-rensa-1 in get sauce installation folder
2. see error

**Expected behavior**
It should download the video. Not sure if it is an issue with the website or it can be bypassed somehow.
Someone who says he is the owner posted in F95 zone allegedly supporting the use of the program to download from his website: https://f95zone.to/threads/ripping-hstream-moe-4k-streams.203201
If it is true, then it might not be caused by web security. half of the videos from the website download without issues.

**Comment by gan-of-culture** [comment]
Hi,

thanks for posting this issue.
Unfortunately I am unable to reproduce the error. Downloading works just fine for me.
There might be a chance that the site has some sort of region blocking or rate-limiting active.
Please retry it again and let me know if you are still getting the same error. If so please try to reach the URL which is causing the EOF error and post it's contents here.

```
get-sauce -w 4 "https://hstream.moe/hentai/kansen-inyoku-no-rensa-1"

 Site:      https://hstream.moe/
 Title:     Kansen Inyoku no Rensa - 1
 Type:      video
 Stream:

     [0]  -------------------
     Type:            video
     Info:            av01.0.12M.08
     Quality:         3840x2160
     Parts:           171
     Size:            0 B
     # download with: get-sauce -s 0 ...

Merging into Kansen Inyoku no Rensa - 1.mp4 ... 100% |████████████████████████████████████████| (373 it/s)
 Site:      https://hstream.moe/
 Title:     Kansen Inyoku no Rensa - 1
 Type:      video
 Stream:

     [1]  -------------------
     Type:            audio
     Info:            mp4a.40.2
     Quality:         unknown
     Parts:           171
     Size:            0 B
     # download with: get-sauce -s 1 ...

Merging into Kansen Inyoku no Rensa - 1.aac ... 100% |████████████████████████████████████████| (4925 it/s)
Merging files using ffmpeg...
[-y -i /home/anon/Kansen Inyoku no Rensa - 1.mp4 -i /home/blank/Kansen Inyoku no Rensa - 1.aac -c copy Kansen Inyoku no Rensa - 1_merged.mp4]
Success!
```

**Comment by Anonym121** [comment]
Hello, I have tested with the same arguments you used (-w 4) and I still get the same error.
I have also tried Ultrasurf and Psiphon VPNs and set them to countries like nederlands, letonia or singapore,; but I still have the same issue. I also tried cmd in admin mode; but it didnt change anything.
I also tried using my phone wifi since I was using my cable internet before; but still no changes.
As I said in my first post it is really strange because I can download half of the videos from the site and the other half give me this "EOF" error.
I can access the url just fine in any of my browsers though.
The url is a direct download link to manifest.mpd
I'll attach it here as manifest.txt to be able to upload it.
[manifest.txt](https://github.com/user-attachments/files/16677773/manifest.txt)
